index.html

```diff
@@ -28,7 +28,7 @@ <h1><h1>I'M Saba Ahmad <br> Glad to see <br> you!</h1></h1>
     
       <h3>LET'S CONNECT</h3>
     </div>
-      <div class="socials">
+      <div class="social">
```

Comment:
- [ ] For semantic reasons, kindly make use of `<ul>` and `<li>` tags for the list of social icons. e.g
``` markup
   <ul>
        <li>
            <a>social icon</a>
        </li>
          ...
   </ul>
```